aten/src/ATen/Context.cpp

```diff
@@ -44,6 +44,14 @@ void Context::setUserEnabledMkldnn(bool e) {
   enabled_mkldnn = e;
 }
 
+bool Context::allowBF32MkldnnMatmul() const {
+  return float32_matmul_precision == at::Float32MatmulPrecision::MEDIUM;
+}
+
+void Context::setAllowBF32MkldnnMatmul(bool b) {
+  float32_matmul_precision = b ? at::Float32MatmulPrecision::MEDIUM : float32_matmul_precision;
+}
```

Comment:
Can we reuse `setFloat32MatmulPrecision` and `float32MatmulPrecision` without introducing new backend functions? Also suggest not to introduce the new "bf32" concept to the end users.

Reply:
Hi, @jgong5. This is following `CuBLAS`.
`CuBLAS TF32` can be enabled by either `setFloat32MatmulPrecision('high')` or `torch._C._set_cublas_allow_tf32(value)`.
So I make `MKLDNN BF32` can be enabled can be enabled by either `setFloat32MatmulPrecision('medium')` or `torch._C._set_mkldnn_bf32_matmul_enabled(True)`.

If `BF32` is not suggest to introduced to end users like `TF32`, I can remove `torch._C._set_mkldnn_bf32_matmul_enabled(True)`. and only use `setFloat32MatmulPrecision('medium')`.
Do you have more preference on it?

Reply:
> and only use `setFloat32MatmulPrecision('medium')`.
> Do you have more preference for it?

Yes, using `setFloat32MatmulPrecision('medium')` sounds better.

Reply:
Hi, @jgong5. I have removed `mkldnn_bf32_matmul_enabled` `get/set`